src/types.js

```diff
@@ -3,7 +3,7 @@ type zeroThroughEleven = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11
 
 export type Offset = zeroThroughEleven | void
 
-export type Size = zeroThroughEleven | 'fit' | 'auto' | void
+export type Size = zeroThroughEleven | 12 | 'fit' | 'auto' | void
```

Comment:
I know we had discussed removing `12` as an option, but it's technically a valid prop value and is causing prop-types to invalidate when storybook knobs is passing `12`. The other option would be to have the storybook knobs range from 1 to 11, or undefined (if thats possible).